src/raftstore/store/worker/region.rs

```diff
@@ -279,12 +279,18 @@ impl SnapContext {
         let end_key = keys::enc_end_key(&region);
         check_abort(&abort)?;
         self.cleanup_overlap_ranges(&start_key, &end_key);
-        box_try!(util::delete_all_in_range(
+        let need_flush_mem_table = box_try!(util::delete_all_in_range(
             &self.engines.kv,
             &start_key,
             &end_key,
             self.use_delete_range
         ));
+        for (cf, need_flush) in self.engines.kv.cf_names().iter().zip(need_flush_mem_table) {
```

Comment:
How about `need_flush_mem_table` is vec![]?

Reply:
So the for loop will be empty.